### PR TITLE
first pass at getting a net developer role

### PR DESCRIPTION
**Pull request by bjwest55**
Not exhaustive at all but a start

### CODE REVIEW

File: roles/engineering-net-developer.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+You enjoy working with others and helping mentor developers at all skill levels.  You are a strong communicator who takes the time to interact with people across different disciplines. You have a keen sense of community and are always looking for new ways to expand the work of others.
+
+<what-you-need.snippet/>
+
+## Skills that are important to this role
+
+<skills>
+Coding
+Language C Sharp:1
+Ide Rider
```
Comment (codeallthethingz) [inline roles/engineering-net-developer.md:28]:
```suggestion
IDE Rider
```

File: roles/engineering-net-developer.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Linux
+QA Automation
+Fire Handling
+Docker
+Machine Learning
+Big Data
+Athena
+Customer Empathy
+Part of Successful Projects:1
+Prometheus
```
Comment (codeallthethingz) [inline roles/engineering-net-developer.md:42]: I feel like a bunch of these are common between developer and net developer and should be moved into an eng-full-stack-base role?

File: competencies/azure-portal.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+## How do you prove it
+* Show you can navigate around the azure portal
+* Show an understanding of the different pieces of info that live in the azure portal
+* Demonstrate ability to change app settings for different projects
+* Show an understanding of the cost management tool
+
+## How do you improve it?
+* Ask for a tour of the portal
+* Poke around
```
Comment (codeallthethingz) [inline competencies/azure-portal.md:13]: I would make these more concrete things that someone could do on their own.

* ask for a tour / poke around -> spend time in the portal and read the help sections

File: competencies/language-c-sharp.md [hunk 4]
```diff
@@ -1,27 +1,29 @@
...
+* You can describe the kinds of problems the language solves very well and the kinds of problems it's not a great fit for.
+* You can write complex LINQ statements without having Rider transform your for loops into LINQ
+* Your code is memory efficient and takes advantage of lazy code execution
 
 ## How do you improve it?
-
-Do the tutorials: https://docs.microsoft.com/en-us/dotnet/csharp/
-
-Build a program in C#
+* Open many PRs
```
Comment (codeallthethingz) [inline competencies/language-c-sharp.md:29]: How many is many?   Good to put a number in.

**Review by codeallthethingz — CHANGES_REQUESTED** [review]
Awesome work!  ❤️ 

Couple of changes requested.

File: roles/engineering-net-developer.md (hunk 2 above)
Comment (bjwest55) [reply roles/engineering-net-developer.md]: Done

**Review by codeallthethingz — APPROVED** [review]

**Pull request merged**